src/app/shared/model/base.model.ts

```diff
@@ -126,13 +128,34 @@ export abstract class BaseModel {
         });
         this.relatedReplays[relName] = relValue;
       }
-      return this.relatedReplays[relName];
+      return this.relatedReplays[relName].first();
```

Comment:
This was driving me nuts.  Always call `first()` here so that the observable completes, and the `forkJoin` methods don't just hang forever.